build-docker-plotman.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+DOCKER_REGISTRY="graemes"
+LOCAL_REGISTRY="registry.graemes.com/graemes"
```

Comment:
We probably don't want to reference your personal stuff here?  At least not unless we are going to host it there for everyone else to use?